src/cry/command.cr

```diff
@@ -43,18 +57,29 @@ module Cry
       else
         Dir.mkdir("tmp") unless Dir.exists?("tmp")
 
-        unless args.code.blank? || File.exists?(args.code)
-          File.write(@filename, "puts (#{args.code}).inspect")
-        else
-          prepare_file
-          system("#{options.editor} #{@filename}")
-        end
+        loop do
+          unless args.code.blank? || File.exists?(args.code)
+            File.write(@filename, "puts (#{args.code}).inspect")
+          else
+            prepare_file
+            system("#{options.editor} #{@filename}")
+          end
+
+          break unless File.exists?(@filename)
 
-        result = ""
-        result = `crystal eval 'require "#{@filename}";'` if File.exists?(@filename)
+          result = ""
+          result = `crystal eval 'require "#{@filename}";'`
 
-        File.write(@filename.sub("console.cr", "console_result.log"), result) unless result.blank?
-        puts result
+          File.write(@result_filename, result) unless result.nil?
+          puts result
+
+          break unless args.loop?
+          puts "\nENTER to edit, q to quit"
+          input = gets
+          break if input=~ /^q/i
+          @filename = new_file
```

Comment:
Just set `@filename_seed = Time.now.epoch_ms` instead of @filename and @result_filename